### PR TITLE
add new company - Bit Zesty

### PR DESCRIPTION
**Pull request by matthewford**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: 

- [x] I am an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/main/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ Name | Website | Region
 [Big Cartel](/company-profiles/big-cartel.md) | https://www.bigcartel.com | USA
 [Big Wheel Brigade](/company-profiles/big-wheel-brigade.md) | https://www.bigwheelbrigade.com/ | USA
 [Bill](/company-profiles/bill.md) | https://www.bill.com/ | USA
+[Bit Zesty](/company-profiles/bit-zesty.md) | https://bitzesty.com/ | UK & Worldwide
```
Comment (dougaitken) [inline README.md:87]: Change to Worldwide to match profile.

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (matthewford) [reply README.md]: @dougaitken sure thing

**Comment by dougaitken** [comment]
@matthewford Why have you removed CareCru? I think it might be due to the force push. Can you check?

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Thanks @matthewford! Merged.

**Pull request merged**